packages/backend-modules/auth/migrations/sqls/20241220130020-removeTwitterFacebook-down.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE users
+ADD COLUMN "publicUrl" text;
+ADD COLUMN "facebookId" text;
+ADD COLUMN "publicUrl" text;
```

Comment:
The SQL syntax here is wrong. `;` ends a statement so the `ADD COLUMN` after `publicUrl` will fail.

The lines should be delimited by `,`

Reply:
done